.github/workflows/playwright-e2e.yml

```
@@ -0,0 +1,139 @@
+
+jobs:
+  path-filter:
+    runs-on: ubuntu-latest
+    outputs:
+      not-ci: ${{ steps.filter.outputs.not-ci }}
+    steps:
+      - name: Checkout for path filter
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.pr != 0 && format('refs/pull/{0}/merge', inputs.pr) || (inputs.branch != '' && inputs.branch || github.sha) }}
+      - uses: dorny/paths-filter@v2
```

actionlint (1.7.12)

[error] the runner of "dorny/paths-filter@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)